lib/LaTeXML/Post/MathML/OperatorDictionary.pm

```diff
@@ -0,0 +1,248 @@
+# /=====================================================================\ #
+# |  LaTeXML::Post::MathML::OperatorDictionary                          | #
+# | MathML generator for LaTeXML                                        | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+
+package LaTeXML::Post::MathML::OperatorDictionary;
+use strict;
+use warnings;
+use LaTeXML::Common::Error;
+use LaTeXML::Util::Unicode;
+use base qw(Exporter);
+our @EXPORT = (qw( &opdict_lookup));
+
+#======================================================================
+# Algorithm loosely derived from MathML Core 2.6.1
+
+# Special Tables defined at end
+our $Operators_2_ascii_chars;
+our $Operators_fence;
+our $Operators_separator;
+our $Content_form;
+our $Category_data;
+# Spacing constants, defined at end
+our $thin;     #  = 0.167;    # 3 mu in ems
+our $med;      #   = 0.222;    # 4 mu
+our $thick;    # = 0.278;    # 5 mu
+
+# Rough assumption of what MathML form each role corresponds to
+# (without actually looking at its position within the expression!)
```

Comment:
Indeed, that lack of contextual positioning is bound to crop up in the future -- great to have the comment here. 

Seeing `VERTBAR` as `postfix` is a notable example.

Reply:
In the long run, I think a good strategy is having `role` indicate the (possibly ambiguous) grammatical roles going into parsing, and have them come out of parsing being tweaked to (notationally) unambiguous roles.  Hopefully, the unambiguous grammatical role is a sufficiently direct mapping to presentational display atom-types or forms.